### DISCUSSION TITLE
no prompt

### DISCUSSION THREAD

**Discussion by petrope** [jgm/pandoc#10122] [General]
Hi,

I have installed  pandoc 3.3, with no success.
I cannot identify what went wrong.

Installed pandoc on win10 with msi.
Read the installing manual. 
Set up path variable.

Read the manual.

I can run oandoc with exe, but without prompt.
I can write in the window pandoc --version, enter, and cursor feed line, and ctrl+C to leave.

How can I debug my install?

Thanks
petrope

**Comment by jgm** [comment]
When using pandoc in the Windows command prompt, use ctrl+Z, not ctrl+C, to terminate your input.
